js/helpers/package.json

```diff
@@ -31,7 +31,7 @@
   "private": true,
   "scripts": {
     "build": "pnpm run build:dev",
-    "build:dev": "tsc --build --incremental .",
```

Comment:
Was this causing issues?

Reply:
Yeah it often fails to build with incremental on. Should've fixed in an other PR, ended up committing here by accident